common/scala/src/main/scala/org/apache/openwhisk/core/database/cosmosdb/CosmosDBArtifactStore.scala

```diff
@@ -21,6 +21,7 @@ import java.io.ByteArrayInputStream
 
 import _root_.rx.RxReactiveStreams
 import akka.actor.ActorSystem
+import akka.event.slf4j.SLF4JLogging
```

Comment:
Is there any reason this cannot use the existing `org.apache.openwhisk.common.Logging` instead of `SLF4JLogging`?

Reply:
Makes sense. Switched to default logging